Package.swift

```diff
@@ -31,6 +31,10 @@ let package = Package(
         .library(
             name: "FraudProtection",
             targets: ["FraudProtection", "PPRiskMagnes"]
+        ),
+        .library(
```

Comment:
nit pick: do we want to move this above the Magnes library?
Looks like all the payment method libraries are together and then Magnes.
Otherwise it looks good!

Reply:
Looks like you have to resolve this:
/Users/runner/work/paypal-ios/paypal-ios/Sources/VenmoPayments/VenmoClient.swift:4:1: error: Vertical Whitespace before Closing Braces Violation: Don't include vertical whitespace (empty line) before closing braces (vertical_whitespace_closing_braces)